docusaurus-docs/jupyterlab-conda-store/introduction.md

```diff
@@ -1,7 +1,71 @@
 ---
-sidebar_position: 1
 title: Introduction
 description: Introduction to JupyterLab Extension.
 ---
 
 # conda-store JupyterLab extension
+
+A extension to use the [conda-store UI][conda-store-ui] - a React-based frontend for conda-store, within JupyterLab.
+
+## Install 📦
+
+1. Pre-requisites: `conda-store-server`, JupyterLab `>=` 3.0 and `<=` 4.0, and Python `>=` 3.8 installed.
+
+2. Install the extension:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
```

Comment:
Oh neat. What are these `<Tabs>` and `<TabItem>`? Are these a MUI thing?

Reply:
I'm not sure how these are implemented. Docusaurus provides this by default, ref docs: https://docusaurus.io/docs/markdown-features/tabs#displaying-a-default-tab